utils/diff_output_utils.go

```diff
@@ -0,0 +1,66 @@
+package utils
+
+type DiffResult interface {
+	GetStruct() DiffResult
+	OutputText(diffType string) error
+}
+
+type MultiPackageDiffResult struct {
```

Comment:
I think I just omitted the "Version" part to shorten it because it seemed really long, but I can put it back... SingleVersionDiffResult never existed, I believe you called it PackageDiffResult and it's still there, but for consistency sake I can make it SingleVersionPackageDiffResult.  These all just seem very long.